tests/e2e/TestConstants.ts

```diff
@@ -233,6 +233,21 @@ export const TestConstants = {
     /**
      * Token for a github repository with permissions which allow add the ssh keys
      */
-    TS_GITHUB_TEST_REPO_ACCESS_TOKEN: process.env.TS_GITHUB_TEST_REPO_ACCESS_TOKEN || ''
+    TS_GITHUB_TEST_REPO_ACCESS_TOKEN: process.env.TS_GITHUB_TEST_REPO_ACCESS_TOKEN || '',
+
+    /**
+     * Login for a user whom has been created in the test Openshift cluster. Need for Openshift connector test
+     */
+    TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER: process.env.TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER || '',
```

Comment:
`TS_LOGIN_NAME_OF_OPENSHIFT_REGULAR_USER` looks a little bit confusing, because:
1) it could be any user including admin
2) it doesn't clear that the parameter is for Openshift Connector test only

Therefore I would propose to rename it to `TS_TEST_OPENSHIFT_PLUGIN_USERNAME`.